### PR TITLE
Add name to data to get audiences.

### PR DESCRIPTION
**Pull request by jfkhoury**
## Description

Remove QE edge domain.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ X ] All tests pass and I've made any necessary test changes.
- [ X ] I have run the Sandbox successfully.

### CODE REVIEW

File: sandbox/public/index.html [hunk 1]
```diff
@@ -71,7 +71,8 @@
           }
         },
         data: {
-          nonXdmKey: "nonXdmValue"
+          nonXdmKey: "nonXdmValue",
+          name: "home"
```
Comment (Aaronius) [inline sandbox/public/index.html]: It might be good to add a comment here as to the importance of `name`.

Comment (jonsnyder) [inline sandbox/public/index.html]: Can you add a comment that this key/value is for Audience Manager?

Comment (jfkhoury) [reply sandbox/public/index.html]: Done @jonsnyder

Comment (jonsnyder) [reply sandbox/public/index.html]: Jinx!

**Pull request merged**